README.md

```diff
@@ -43,12 +44,11 @@ This will start `cargo-marker`, load [`./marker_lints`](./marker_lints) as a *li
 
 Internal structure of marker and its components is documented in [`docs/internal/`](./docs/internal/).
 
-## License
+---
 
-Copyright (c) 2022-2022 Rust-marker
-
-Rust-marker is distributed under the terms of both the MIT license
-and the Apache License (Version 2.0).
-
-See [LICENSE-APACHE](./LICENSE-APACHE), [LICENSE-MIT](./LICENSE-MIT).
+#### License
```

Comment:
I'd prefer this to be a normal section with normal text size. It makes the file more uniform and this is an equal level as the other parts IMO. Is there a reason why you suggest this?

Reply:
I stated the reason in the second comment, copy-pasting :sweat_smile:

Going to rollback that change right now anyways